src/cal_build/traceExtract_GH.jl

```diff
@@ -796,11 +809,9 @@ function trace_extract(image_data, ivar_image, tele, mjd, chip, expid,
     curr_fiber_inds = clamp.(range(1, size(best_fit_ave_params, 1)), min_prof_fib, max_prof_fib)
 
     med_flux = nanmedian(best_fit_ave_params[:, 1], 1)
-    good_throughput_fibers = (best_fit_ave_params[:, 1] ./ med_flux) .> 0.1
+    good_throughput_fibers = (best_fit_ave_params[:, 1] ./ med_flux) .> 0.05
```

Comment:
I would like to voice that making these keyword arguments, if they are parameters we are going to be tweaking for stability, would make me happier